test/engine/network/index.test.ts

```diff
@@ -1,31 +1,23 @@
-// import { describe, it } from "vitest";
 import { ok, strictEqual } from "assert";
```

Comment:
What happens with these network tests after we've removed the old network messages? Is this no longer needed?